src/to_geojson.rs

```diff
@@ -63,10 +61,7 @@ fn make_feature_id(id: Option<JsonValue>) -> Option<FeatureId> {
     match id {
         Some(JsonValue::Number(i)) => Some(FeatureId::Number(i)),
         Some(JsonValue::String(i)) => Some(FeatureId::String(i)),
-        _ => {
```

Comment:
The alternate proposal: `Some(_) => log::warn` and `None => None` without any logging